app/templates/security/_menu_new.html

```diff
@@ -5,8 +5,5 @@ <h2 class="sr-only">{{ gettext('Menu') }}</h2>
   {% if security.recoverable %}
   <li><a href="{{ url_for_security('forgot_password') }}">{{ gettext('Forgot password') }}</a><br/></li>
   {% endif %}
-  {% if security.confirmable %}
-  <li><a href="{{ url_for_security('send_confirmation') }}">{{ gettext('Confirm account') }}</a></li>
-  {% endif %}
```

Comment:
Since email confirmation is not *required* to login to the site, there's no reason to feature it on the security links that appear at the bottom of security pages.